Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -608,33 +608,41 @@ public static List<TreeViewItemData<ActionOrBindingData>> GetActionsAsTreeViewDa
                 {
                     var serializedInputBinding = actionBindings[i];
                     var inputBindingId = new Guid(serializedInputBinding.id);
-
+                    
```

Comment:
It seems these changes prevent any exception from being thrown which is an improvement so it could land in that sense but I think we should instead prevent the last instance of each composite part being supported by the composite class from being deleted. If we do not prevent this we have a non-reversible behavior in the editor where deleting a composite part cannot be re-added. We allow duplicating the parts which is fine I guess for adding multiple bindings per part but I believe we should avoid deleting the last instance of each part.